relentless/simulate/hoomd.py

```diff
@@ -198,7 +198,7 @@ class InitializeFromFile(Initialize):
     neighbor_buffer : float
         Buffer width.
     options : kwargs
-        Options for file reading (as used in :py:func:`hoomd.init.read_gsd()`).
+        Options for file reading (as used in :py:meth:`hoomd.init.read_gsd()`).
```

Comment:
Did this cause a compile error? `read_gsd` is a function not a class method in HOOMD 2.9.x. If it is causing issues, we might need to change the intersphinx url so it finds the right version on RTD.

Actually, maybe we haven't linked HOOMD into the intersphinx docs yet? Let's make a note to do that.